documentation/projects/proposals/rekognition_data/20240320-project_proposal_rekognition_data.md

```diff
@@ -0,0 +1,221 @@
+consider how these machine-generated tags are displayed and whether they can be
+interacted with in the same way. Similar to the API, it may also be useful to
+share the label accuracy with users (either visually or with extra content on
+mouse hover). It would be beneficial to have a page much like our
+[sensitive content explanation](https://openverse.org/sensitive-content) (either
+similarly available in the frontend or on our documentation website) that
+describes the nature of the machine generated labels, the means by which they
+were determined, and how to report an insensitive label.
+
+None of the above is specific to Rekognition, but it will be necessary to
+determine for Rekognition or any other labels we wish to add in the future.
+
+### Augmenting the catalog
+
+Once we have a clear sense of how the labels will be shared downstream, we can
+incorporate the labels themselves into the catalog database. This can be broken
+down into three steps:
+
+1. Determine which labels to use (see
+   [label determination](#label-determination))
+2. Determine an accuracy cutoff value
+3. Upsert the filtered labels into the database
+
+Once step 3 is performed, the next data refresh will make the tags available in
+the API and the frontend. The specifics for each step will be determined in the
+implementation plan for this piece. Note that once introduced, the tags will not
+be removed by subsequent updates to the catalog data. This means that any
+adjustment/removal of the tags will also need to occur on the catalog.
+
+#### Label determination
+
+The exhaustive list of AWS Rekognition labels can be downloaded here:
+[AWS Rekognition Labels](https://docs.aws.amazon.com/rekognition/latest/dg/samples/AmazonRekognitionLabels_v3.0.zip).
+While this list is already fairly demographically neutral, it is my opinion that
+we should exclude labels that have a demographic context in the following
+categories:
+
+- Age
+- Gender
+- Sexual orientation
+- Nationality
+- Race
+
+These seem the most likely to result in an incorrect or insensitive label (e.g.
+gender assumption of an individual in a photo). There are other categories which
+might be useful for search relevancy and are less likely to be applied in an
+insensitive manner. Some examples include:
+
+- Occupation
+- Marital status
+- Health and disability status
+- Political affiliation or preference
+- Religious affiliation or preference
+
+Specifics for how this will be tackled regarding the Rekognition data will be
+outlined in the associated implementation plan.
+
+## Success
+
+<!-- How do we measure the success of the project? How do we know our ideas worked? -->
+
+This project can be marked as success once the machine-generated tags from
+Rekognition are available in both the API and the frontend.
+
+If the labels themselves are observed to have a negative impact on search
+relevancy, we will need a mechanism or plan for the API for suppressing or
```

Comment:
I would like to see an acknowledgment in this section about the lack of tools to measure search relevancy. To be clear, I do _not_ think that we should hold off until the Measuring Search Relevancy project is completed to implement this and I don't think that should be a blocker. But I do think it's very important that the Project Proposal captures this discrepancy and explains what our reasoning for going forward with the project without it is. Your comment in an earlier thread using iNauralist as an example is a perfect explanation, IMO :)